native/src/parser.js

```diff
@@ -28,15 +28,14 @@ const GUESSING_ORDER = [
 ];
 
 export function guessParsing(code) {
+  let exceptions = [];
   for(let [fn, opts] of GUESSING_ORDER) {
     try {
       return fn(code, opts);
     } catch (ex) {
-      continue;
+      exceptions.push(ex)
     }
   }
+  throw new Error(exceptions.map((x) => x.message).join(","));
```

Comment:
Yes... my point is... if you captured many Exceptions... why the response emit only one? (one string with messages separated by commas)